Accessories/ExpansionBoard_uInverter/REV20220329B/gerbers/PCB_uinv-RectHoles.TXT

```diff
@@ -0,0 +1,20 @@
+M48
```

Comment:
Is it standard to include a file for rectangular holes like this?

I do not have experience specifying slots and rect holes, so I do not know.

@GnomeTek thoughts?

Reply:
I have pushed both files for now.